src/components/Home/PresentRanking/Item/RankingSexTagItem.tsx

```diff
@@ -21,30 +25,33 @@ const StyledRankingSexTagItem = styled.div`
   justify-content: center;
   margin: 10px;
 `;
+const isValidSelectedOption = (key: string): key is TARGET_ITEM_TYPE => {
+  return TARGET_ITEM_LIST[key] !== undefined;
+};
```

Comment:
타입 가드 함수도 잘 작성해주셨네요 👍
in을 활용해 키가 객체에 있는지 확인하도록 할 수 있어요

```suggestion
const isValidSelectedOption = (key: string): key is TARGET_ITEM_TYPE => {
  return key in TARGET_ITEM_LIST
};
```